src/vsr/replica_test.zig

```diff
@@ -1696,10 +1696,13 @@ test "Cluster: view_change: DVC header doesn't match current header in journal"
     try expectEqual(t.replica(.R0).commit_max(), checkpoint_2_prepare_max);
 }
 
-test "Cluster: view_change: lagging replica repairs WAL using start_view from potential primary" {
+test "Cluster: view_change: lagging replica state syncs using start_view from potential primary" {
     // It could be the case that the replica with the most advanced checkpoint has a corruption in
     // its grid. In this case, a replica on an older checkpoint can use a start_view message from
-    // the most up-to-date replica to repair its WAL, advance its checkpoint, and become primary.
+    // the most up-to-date replica to state sync, advance its checkpoint, and become primary. We
+    // make one core assumption about the corrupted grid block that is preventing the up-to-date
+    // replica from becoming primary -- it is a block that the lagging replica *already* has,
```

Comment:
I am not very happy with this assumption... However, since don't have any granular control over corrupting specific blocks that a replica needs for compaction, I wasn't able to come up with a better way to encode our expectation except through a comment. @sentientwaffle thoughts?